### DISCUSSION TITLE
PFCC 许愿屋

### DISCUSSION THREAD

**Discussion by luotao1** [PaddlePaddle/Paddle#51802] [General]
[Paddle Framework Contributor Club (PFCC)](https://github.com/PaddlePaddle/community/tree/master/pfcc) 是一个有兴趣、正在、或者已经为 [飞桨开源框架](https://github.com/PaddlePaddle/Paddle/) 做开源贡献的贡献者成立的虚拟组织，在这里，飞桨开源框架的贡献者进行讨论、交流和分享，并为飞桨框架做出持续的贡献。

PFCC 许愿屋：有什么梦想，大家一起来实现。 https://github.com/PFCCLab

**Comment by luotao1** [comment]
帮花花做一个自动更新榜单的工具：
- 现在每天在 issue 下认领再更新再统计太累了，pr merge 也不知道
- 不在github 页面上认领和更新也可以，需求就是能认领能提交作品能更新榜单就行

**Reply by gouzil** [reply]
许愿成功🎉🎉: https://github.com/gouzil/github-api-issues

**Comment by sanbuphy** [comment]
利用这个看板https://ost.51cto.com/posts/14589 做一个paddle版本的

**Comment by sanbuphy** [comment]
有一个paddle小助手，监控开源项目进度，及时反馈，可以在微信群互动。@Liyulingyue

**Reply by Liyulingyue** [reply]
创建一个新的文件夹！欢迎大家一起来贡献！

https://github.com/PFCCLab/PaddleAutoProject/tree/main/WeChatBot

这个项目希望建设初期可以投入到快乐开源群的使用中！

对快乐开源活动感兴趣的小伙伴欢迎联系 @krystalxxf 加入群聊~

**Comment by Liyulingyue** [comment]
开发一个带有界面的工具，可以通过Aistudio sdk调用文心一言，以获取一段时间内，指定用户的最具有价值的PR号

**Reply by GreatV** [reply]
什么样的PR算最有价值的PR？

**Reply by Liyulingyue** [reply]
我不知道，最有价值的PR应该像是游戏中或者球赛中的黄金时刻一样。

虽然大众广泛地将很多进球时刻看作是黄金时刻。但是我们依然可以将一些传球、拦截、滑铲、守门地时刻看作是黄金时刻。

因此，在这里，我只能给出一个很宽泛的概念“最具有价值的PR”，但是很难给出具体的定义。在开发这个工具的时候，我们可以考虑让LLM给出黄金PR的定义，并依靠这个定义去寻找对应的PR。

**Reply by gouzil** [reply]
任务简介，修改文件范围，代码行数应该都可以划分权重吧

**Reply by Liyulingyue** [reply]
是的，但是这里其实涉及到几个核心问题
1. LLM读代码
2. LLM理解代码改动的核心程度或者难度
3. LLM理解代码对其他模块的影响程度
